compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -178,9 +178,10 @@ public class SymbolTable {
     public final BRegexpType regExpType = new BRegexpType(TypeTags.REGEXP, Names.REGEXP_TYPE);
     public final BType xmlNeverType = new BXMLType(neverType,  null);
     public final BType xmlElementSeqType = new BXMLType(xmlElementType, null);
+    public final BType xmlItemType = BUnionType.create(null, xmlElementType, xmlCommentType,
```

Comment:
We have created this type within the `Types` class named `expandedXMLBuiltinSubtypes`. Should we remove it?